### DISCUSSION TITLE
install rauc bundle from exfat?

### DISCUSSION THREAD

**Discussion by bg-mt** [rauc/rauc#1096] [General]
Hello
Its seems very trivial, so I'm a bit confused:
Why it is not supported to install a rauc bundle stored on a mounted exfat filesystem?

```
$rauc install XYZ.rauc
installing
  0% Installing
  0% Determining slot states
 20% Determining slot states done.
 20% Checking bundle
 40% Checking bundle failed.
100% Installing failed.
LastError: bundle is stored on an unknown filesystem (type=2011bab0)
```

Is there any specific reason not supporting exfat? (security issues? exfat driver available starting in kernel 5.7 only? )

Best Regards

**Comment by ejoerns** [comment]
@bg-mt We had to make an explicit list to ensure we know where the bundle comes from as part of the mitigation for https://github.com/rauc/rauc/security/advisories/GHSA-cgf3-h62j-w9vv

That you run into this should mean you are using the legacy 'plain' bundle format. Please consider switching to 'verity' bundle format (`format=verity` in the manifest) where the mitigation is not required.

 https://rauc.readthedocs.io/en/latest/reference.html#sec-ref-formats

**Reply by ejoerns** [reply]
Just had another look and since exFAT, like FAT, is also not capable of storing unix-like permissions, adding it to the explicit list would not work.
However, as for FAT you should be able to set a proper `fmask` during mounting to prevent unauthorized access (as noted in https://rauc.readthedocs.io/en/latest/integration.html#bundle-format-migration). But 'verity' is still the primary choice :smirk:
